.github/workflows/ci.yml

```diff
@@ -31,8 +31,15 @@ jobs:
   # both terminal and file outputs
   dte_coordinator:
     name: DTE Coordinator
-    uses: nrwl/ci/.github/workflows/nx-cloud-main.yml@v0.4
+    uses: nrwl/ci/.github/workflows/nx-cloud-main.yml@v0.7
     with:
+      # Must match dte_agents.number-of-agents
+      number-of-agents: 4
+      # The --stop-agents-after parameter must match the task target listed last
+      # in the parallel-commands-on-agents command list
+      # The --agent-count parameter must match the number-of-agents parameter
+      init-commands: |
+        yarn nx-cloud start-ci-run --stop-agents-after="e2e" --agent-count=4
```

Comment:
# --stop-agents-after
Specifying this parameter stops idle agents once all tasks have been distributed. This is useful if one or more agents are processing long-running tasks such as `e2e` targets, allowing other agents to shut down, thereby saving compute time.

See https://nx.dev/nx-cloud/set-up/set-up-dte#--stop-agents-after.

# --agent-count=4
This parameter is optional but optimizes Nx Cloud's scheduling.

The same goes for the `number-of-agents` parameter specified in the reusable DTE coordinator workflow. This is optimizing for edge cases such as many runners and many tasks though, neither of which we have.

Since we cannot use environment variables as the single source of truth for the number of DTE agents as per https://github.com/ngworker/lumberjack-next/pull/43#discussion_r990848072, I think I will remove the optional agent # parameters.

Reply:
praise: Thanks! That makes it very clear.

Reply:
I guess we could use a repository secret to specify and match the number of agents across the 3 parameters. We would have to document this secret and migrate it to https://github.com/ngworker/lumberjack and eventually https://github.com/ngworker/ngworker though.

Reply:
That could be 🤔 . I'm not 100% sold on the idea, especially because it is only repeated twice.

Would it be worth reducing the transparency of the code (hiding the number of workers used) to remove a one-time duplication?

Reply:
What do you think, @NachoVazquez?

Reply:
Let's keep it as is for now. We can discuss it later.